cmd/hub/mcscontroller/controller.go

```diff
@@ -118,15 +121,205 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 //	 automatically deleted by k8s when their ServiceExport is deleted.
 //
 func (r *ServiceExportReconciler) reconcile(ctx context.Context, name types.NamespacedName) (ctrl.Result, error) {
-	_ = r.Log.WithValues("service", name)
+	log := r.Log.WithValues("service", name)
+	//1:
+	var exportList mcsv1a1.ServiceExportList
+	if err := r.Client.List(ctx, &exportList, client.MatchingLabels{"name": name.Name, "namespace": name.Namespace}); err != nil {
```

Comment:
I think this should be lighthouse.submariner.io/sourceName and sourceNamespace

Reply:
reconcile gets called in line 90 of Reconcile with original name and namespace which are extracted from the labels.

Reply:
Yes, but the query match label keys need to be "lighthouse.submariner.io/sourceName" and "lighthouse.submariner.io/sourceNamespace"

Reply:
Correct. 
@itay-nakash there's `lhutil.ServiceExportListFilter` that does the list options filter for you